acceptance/tests/resource/package/does_not_exist.rb

```diff
@@ -6,15 +6,11 @@
                          # actual changing of resources could irreparably damage a
                          # host running this, or require special permissions.
 
-  resource_declaration_regex = %r@package \{ 'not-installed-on-this-host':
-  ensure => '(?:purged|absent)',
-\}@m
-
   agents.each do |agent|
 
     step "test puppet resource package" do
       on(agent, puppet('resource', 'package', 'not-installed-on-this-host')) do
-        assert_match(resource_declaration_regex, stdout)
+        assert_match(/package.*not-installed-on-this-host.*\n.*ensure.*(?:absent|purged).*\n.*provider/, stdout)
```

Comment:
Figured adding the provider here would also be a free acceptance test for the changes here.